Misc/NEWS.d/next/C API/2020-04-11-06-12-44.bpo-40170.cmM9oK.rst

```diff
@@ -0,0 +1,2 @@
+Convert :c:func:`PyObject_IS_GC` macro to a function to hide
+implementation details: the macro test an object have GC head or not.
```

Comment:
I would prefer to avoid "GC head" in the code. Here I don't think that it's worth it to repeat the documentation, so maybe just truncate the sentence:

```suggestion
implementation details.
```

Reply:
Thanks, done.